editor/editor_interface.cpp

```diff
@@ -402,6 +406,8 @@ void EditorInterface::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("popup_dialog_centered_ratio", "dialog", "ratio"), &EditorInterface::popup_dialog_centered_ratio, DEFVAL(0.8));
 	ClassDB::bind_method(D_METHOD("popup_dialog_centered_clamped", "dialog", "minsize", "fallback_ratio"), &EditorInterface::popup_dialog_centered_clamped, DEFVAL(Size2i()), DEFVAL(0.75));
 
+	ClassDB::bind_method(D_METHOD("set_current_feature_profile", "profile_name"), &EditorInterface::set_current_feature_profile);
```

Comment:
Should we add a getter as well? Only makes sense so you can avoid setting it unnecessarily or for UndoRedo and other kinds of deep integration.